<launch>
  <arg name="files_path" default="/media/sysu/200GB_Other/Dataset/Slam/kitti360/test_0" />
  <arg name="bag_path" default="/media/sysu/200GB_Other/Dataset/Slam/kitti360/test_0" />
  <arg name="calib_path"
    default="$(find stereo_dso)/cams/kitti360/camera0.txt" />
  <arg name="left_topic_name" default="/camera/left/image_raw" />
  <arg name="right_topic_name" default="/camera/right/image_raw" />

  <node name="stereo_dso1" pkg="stereo_dso" type="dso_dataset_ros" output="screen" required="true">
    <param name="files" value="$(arg files_path)" />
    <param name="bag" value="$(arg bag_path)/test_0.bag" />
    <param name="calib" value="$(arg calib_path)" />
    <param name="topic0" value="$(arg left_topic_name)" />
    <param name="topic1" value="$(arg right_topic_name)" />
    <param name="preset" value="0" />
    <param name="mode" value="1" />
    <param name="quiet" value="1" />
    <param name="nomt" value="1" />
  </node>
</launch>
